app/dashboard/templates/github_comments/status_bar.md

```diff
@@ -0,0 +1,11 @@
+{%if status == 'open' %}
+	![Open bounty]({% static 'status/open-copy.svg' %})
+{%elif status == 'started' and started_work.count %}
+	![Started work]({% static 'status/started-copy.svg' %})
+{%elif status == 'started' and not started_work.count %}
+	![Stopped work]({% static 'status/stopped-copy.svg' %})
+{%elif status == 'submitted %}
+	![Submitted work]({% static 'status/submitted-copy.svg' %})
+{%elif status == 'done' %}
```

Comment:
Can we follow the django template patterns `{% if var  %}`

Reply:
@mbeacom I've removed this file for now. 

I was going to add file another issue to turn the status bar back into a template, which allows for the status bar to accurately reflect the status of contests. We could use the template in the function `get_status_header`. Currently the status bar is based on the event_name that is sent to maybe_market_to_github in the notifications module. We could also use the bounty's status to get rid of the `event_name` parameter entirely, and combine different messages into one status message when the contest bounty calls for it. Below I've listed an example scenario:

1. New contest -> 
2. Issue opened message
4. User 1 starts work ->
5. Start work message 1
5. if contest is still open: Issue opened message
6. User 2 starts work ->
7. Start work message 1
7. Start work message 1
8. if contest is still open: Issue opened message
9. User 1 submits work ->
10. Start work message 1
10. Submitted work message 2
...